docs/faq.adoc

```diff
@@ -1,4 +1,4 @@
-= FAQs and Troubleshooting
+= Frequently Asked Questions and Trouble Shooting
```

Comment:
I'd like to keep Troubleshooting as one word. Also, to help with search engines, let's write FAQs as:

Frequently Asked Questions (FAQs)

wdyt?